mmv1/third_party/terraform/services/cloudrunv2/resource_cloud_run_v2_service_test.go.erb

```diff
@@ -666,3 +667,80 @@ resource "google_cloud_run_v2_service" "default" {
 }
 `, context)
 }
+
+<% unless version == "ga" -%>
+func TestAccCloudRunV2Service_cloudrunv2ServiceCustomAudienceUpdate(t *testing.T) {
+	t.Parallel()
+
+  serviceName := fmt.Sprintf("tf-test-cloudrun-service%s", acctest.RandString(t, 10))
+	context := map[string]interface{}{
+		"service_name": serviceName,
+	}
+
+	acctest.VcrTest(t, resource.TestCase{
+		PreCheck:                 func() { acctest.AccTestPreCheck(t) },
+		ProtoV5ProviderFactories: acctest.ProtoV5ProviderBetaFactories(t),
+		CheckDestroy:             testAccCheckCloudRunV2ServiceDestroyProducer(t),
+		Steps: []resource.TestStep{
```

Comment:
We should probably have another step here that changes the custom_audiences value (instead of unsetting it) just to be safe.

Reply:
Absolutely, I can make that happen, I added it so if we can re-run the tests that would be great!